bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -8,164 +8,311 @@
+    <div id="tabzilla">
+      <a href="{{ url('mozorg.home') }}">{{ _('Mozilla') }}</a>
+    </div>
+    {% block site_header_nav %}
+      {% include 'firefox/includes/release-menu.html' %}
+    {% endblock %}
+    {% block site_header_logo %}
+      <h2><a href="{{ url('firefox') }}">{{ high_res_img('firefox/template/header-logo.png', {'alt': 'Mozilla Firefox', 'width': '185', 'height': '70'}) }}</a></h2>
+    {% endblock %}
+  </div>
+</header>
+{% endblock %}
+
 {% block content %}
-<main role="main">
+<main role="main" class="release-notes">
   <article id="main-content">
 
   {% block notes_header %}
     <header class="notes-head">
-    {% block notes_heading_primary %}
-      <h1>{{ _('Firefox {channel} Notes')|f(channel=channel_name) }}</h1>
-    {% endblock %}
-    {% block notes_heading_secondary %}
-      <h2>{{ _('Version {version}, first offered to {channel} channel users on {date}')|f(channel=release.channel, date=release.release_date|l10n_format_date, version=release.version) }}</h2>
-    {% endblock %}
+      <div class="intro">
+        <div class="container">
+        {% block notes_heading_primary %}
+          <h1>{{ _('See what’s new in Firefox!') }}</h1>
+        {% endblock %}
+          <p>
+            {% trans
+            feedback='https://input.mozilla.org/feedback/android/' + release.version + '?utm_source=releasenotes'
+            if release.product == 'Firefox for Android'
+            else 'https://input.mozilla.org/feedback/firefox/' + release.version + '?utm_source=releasenotes',
+            bugzilla='https://bugzilla.mozilla.org/',
+            bug_search_url=release.get_bug_search_url()
+            %}
+              Release Notes tell you what’s new in Firefox. As always, we welcome your <a href="{{ feedback }}">feedback</a>. You can also <a href="{{ bugzilla }}"> file a bug in Bugzilla </a> or see the <a href="{{ bug_search_url }}">complete list of changes</a> in this release.
+            {% endtrans %}
+          </p>
+        </div>
+      </div>
 
     {% block platform_switch %}
-    {% if equivalent_release_url %}
-      <aside id="platform">
-        <h4>{{ _('View notes for:') }}</h4>
-        <ul>
-        {% if release.product == 'Firefox' %}
-          <li><span>{{ _('Desktop') }}</span></li>
-          <li><a href="{{ equivalent_release_url }}">{{ _('Android') }}</a></li>
-        {% else %}
-          <li><a href="{{ equivalent_release_url }}">{{ _('Desktop') }}</a></li>
-          <li><span>{{ _('Android') }}</span></li>
-        {% endif %}
-        </ul>
-      </aside>
-    {% endif %}
+      <div id="nav" class="navigator">
+        <div class="container">
+          <ul class="menu">
+          {% if release.product == 'Firefox' %}
+            <li class="current">{{ _('Desktop') }}</li>
+            <li><a href="{{ url('firefox') + 'android/notes/' }}">{{ _('Android') }}</a></li>
+            <li><a href="{{ url('firefox') + 'ios/1.1/releasenotes' }}">{{ _('iOS') }}</a></li>
```

Comment:
> Should the version number be hard-coded here? iOS notes are still taking shape but it might be better to have a generic URL with a redirect (so the redirect could be updated in one place for new releases), rather than hard-code it into the links (and have to update them in lots of places for each release).

Good question. I'm not sure how best to address this though. Maybe product-details can tell be the #. I'll check.